docs/apis/avalanchego/public-api-server.md

```diff
@@ -288,3 +288,23 @@ Features:
 ##### HTTPS
 
 - For C-Chain API, the URL is `https://avalanche-c-chain.publicnode.com`
+
+### OnFinality 
+
+[OnFinality](https://onfinality.io/networks/avalanche) currently only supports the P-Chain.
+
+#### HTTP
+
+- For P-Chain API, the URL is  `https://avalanche.api.onfinality.io/public/ext/bc/P`
+
+#### Websockets
+
+- For P-Chain API, the URL is  `https://avalanche.api.onfinality.io/public/ext/bc/P`
+
+Features:
+
+- Powerful API Analytics
+- Full Archive Node Access
+- Globally Distributed Infrastructure
+- Enhanced Performance
```

Comment:
This part should not be title case.

Reply:
@deverka can you please check out the above request?